### PR TITLE
Chore: improve log message about cache size limit exceeded

### LINKED ISSUE #114527: [debugging] Given a flag, `FakeTensor`s should store metadata about their creation stacktrace
### 🚀 The feature, motivation and pitch

It's much easier to understand the identity of a problematic fake tensor if it comes with information about its originating node or source

cc @ezyang @msaroufim @wconstab @bdhirsh @anijain2305 @zou3519 @eellison

### PR DESCRIPTION
**Pull request by anabasalo**
Fixes #114527

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

### CODE REVIEW

File: torch/_dynamo/cache_size.py [hunk 1]
```diff
@@ -154,8 +161,12 @@ def is_recompilation(cache_size: CacheSizeRelevantForFrame) -> bool:
     return cache_size.will_compilation_exceed(1)
 
 
-def exceeds_cache_size_limit(cache_size: CacheSizeRelevantForFrame) -> bool:
+def exceeds_cache_size_limit(cache_size: CacheSizeRelevantForFrame) -> Tuple(bool, str):
```
Comment (Skylion007) [inline torch/_dynamo/cache_size.py:164]:
```suggestion
def exceeds_cache_size_limit(cache_size: CacheSizeRelevantForFrame) -> Tuple[bool, str]:
```

Comment (anabasalo) [reply torch/_dynamo/cache_size.py]: @Skylion007 sorry, tks
[done](https://github.com/pytorch/pytorch/pull/116557/commits/ca2f6e337f97e4218959fab729035586c2d09cee)

**Review by ezyang — APPROVED** [review]

**Comment by anabasalo** [comment]
@ezyang there was a problem with the workflow, but i fixed it
tks

**Comment by anabasalo** [comment]
@ezyang finally now i solved all the lint issues.
sorry and tks

**Pull request closed without merging**